src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4729,8 +4729,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RefPropertyCannotHaveSetAccessor" xml:space="preserve">
     <value>Properties which return by reference cannot have set accessors</value>
   </data>
-  <data name="ERR_CantChangeRefReturnOnOverride" xml:space="preserve">
-    <value>'{0}' must {2}return by reference to match overridden member '{1}'</value>
+  <data name="ERR_CantChangeRefnessOfReturnOnOverride" xml:space="preserve">
```

Comment:
Because it might be "cannot change ref readonly return" as well. Need to indicate that this involves other kinds of refness as well.

---
In reply to: [104703333](https://github.com/dotnet/roslyn/pull/17547#discussion_r104703333) [](ancestors = 104703333)